Java/PoolAndResourceFile/pom.xml

```diff
@@ -11,37 +11,13 @@
 
   <groupId>com.microsoft.azure</groupId>
   <artifactId>PoolAndResourceFile</artifactId>
-  <version>1.0-SNAPSHOT</version>
```

Comment:
hmm what is a snapshot?  I guess they are no longer neeeded now that this is transitioning to ga?